learn/getting-started/install-and-connect-harper.mdx

```diff
@@ -0,0 +1,260 @@
+
+The fastest way to get started is by installing Harper globally using an npm compatible package manager:
+
+```bash
+npm install -g harperdb
+```
+
+  </TabItem>
+  <TabItem value="docker" label="Docker">
+
+Harper is readily available as a Docker image [`harperdb/harperdb`](https://hub.docker.com/r/harperdb/harperdb).
+
+The image is based off of a Node.js image and the default tag is always published using the latest Harper version and latest Node.js Active LTS version.
+
+The image uses sensible default environment variables, agreeing to the terms and conditions, setting a rootpath ensured by the image itself, creating a default admin user with username `HDB_ADMIN` and password `password`, and enabled standard streams logging.
+
+Using a Docker compatible container manager of choice, the simplest way to get started is using:
+
+```bash
+docker pull harperdb/harperdb
+docker run -it \
+  --name harper \
+  -v /Users/<user>/hdb:/home/harperdb/hdb \
+  -v /Users/<user>/dev:/home/harperdb/dev \
+  -e DEFAULTS_MODE=dev \
+  -e REPLICATION_HOSTNAME=localhost \
+  -p 9925:9925 \
+  -p 9926:9926 \
+  harperdb/harperdb \
+  /bin/bash
+```
+
+The `-v` options will mount the Harper installation (`hdb/`) as well as a development directory (`dev/`) to the container host which is useful for development purposes. Now the `hdb/` path will contain the Harper installation parts (once you install), and the `dev/` directory can be used to create projects (which future guides will require).
+
+The additional environment variables specified by `-e` options ensures the installation is setup for local development.
+
+Without the `-it` and `/bin/bash` parts, the image will start (and install) the Harper application by default. For the purposes of these guides, access to the `harperdb` CLI is necessary, so we recommend to always exec into the container.
+
+  </TabItem>
+</Tabs>
+
+Then, execute the Harper CLI to start the installation process:
+
+```bash
+harperdb install
+```
+
+The Harper installation process is an interactive prompt, however it also supports overrides using environment variables or CLI arguments. It supports partial or complete overrides for the installation prompts and even allows additional configuration options to be specified too.
+
+This guide focusses on a happy path in order to keep things simple. If you're interested in learning more about all of Harper's configuration options see the the [Harper CLI](../../docs/deployments/harper-cli) reference docs.
+
+<Tabs groupId="local-install">
+  <TabItem value="npm" label="npm">
+
+When installing locally on your machine, you can specify any destination for Harper, we recommend using something within your user directory such as `/User/<user>/hdb`. Keep note of what you specify for the username and password, and make sure you select the `dev` default config and set the hostname to `localhost`.
```

Comment:
I can tell which OS is used by the writer of these docs :). That's probably fine, I guess MacOS probably has the largest share of devs.

Reply:
Yeah can totally find a better way to describe this here. It felt weird having like my personal `/Users/ethan/` in the examples so I was trying to make it general purpose. I don't know what makes most sense.